systemtests/src/test/java/io/enmasse/systemtest/scale/ScaleTest.java

```diff
@@ -372,6 +385,11 @@ public void testFailureRecovery() throws Exception {
 
         } finally {
             saveResultsFile("failure_recovery_results.json", manager.getDowntimeResult());
+            Map<String, Object> data = new HashMap<>();
+            data.put("addresses", addresses.size());
+            data.put("connections", manager.getConnections());
+            data.put("normalTimeToCreateAddress", manager.getDowntimeResult().getNormalTimeToCreateAddress());
+            savePlotCSV("failure_recovery.csv", data);
```

Comment:
all the information related to specific downtime per pod removed is missing, however I understand that could be hard to represent in a csv. I would represent this with one row per pod removed.

Reply:
as I mention above, csv must be single line, so I cant add data like this, I only can list all pods and add them as columns into csv @famartinrh

Reply:
ok, let me think of it, and I could give a summarized result that fits into one line